pkg/apis/openebs.io/v1alpha1/register.go

```diff
@@ -4,8 +4,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-
-	"github.com/openebs/maya/pkg/apis/openebs.io"
```

Comment:
Don't have to remove this ...

Reply:
Sorry, messed up with the reverts. Fixing it.